src/c++/perf_analyzer/docs/README.md

```diff
@@ -48,3 +48,6 @@ model input data, the performance measurement modes, the performance metrics and
 outputs, how to benchmark different servers, and more.
 
 - [Perf Analyzer CLI](cli.md)
+- [Data Guide](data_guide.md)
```

Comment:
I think it would be better to call this `Input Data` (and rename the file to `input_data.md`.

Reason being is that I think for most users, they're concerned with how to supply custom input data. Output validation seems not to be used that much, and seems appropriate to add as a section to the Input Data doc, regardless of the fact that it's not technically Input Data.

Main focus here is making the documentation page name recognizable to user's needs.

Reply:
thats fine with me